polaris-react/src/components/KeyboardKey/KeyboardKey.scss

```diff
@@ -1,23 +1,32 @@
-$key-base-height: 24px;
+$key-base-dimension: 28px;
 
 .KeyboardKey {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: inline-block;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $key-base-height;
+  height: $key-base-dimension;
+  display: inline-flex;
+  justify-content: center;
   margin: 0 var(--p-space-05) var(--p-space-05);
-  padding: 0 var(--p-space-1);
-  background-color: var(--p-surface);
-  box-shadow: 0 0 0 var(--p-space-025) var(--p-border-subdued),
-    0 var(--p-space-05) 0 0 var(--p-surface),
-    0 var(--p-space-05) 0 var(--p-space-025) var(--p-border-subdued);
-  border-radius: var(--p-border-radius-1);
-  font-family: var(--p-font-family-sans);
-  font-size: var(--p-font-size-75);
+  padding: 0 var(--p-space-2);
+  background: var(--p-surface-depressed);
```

Comment:
Just a note on some of the color tokens here. They're not always mapping to their semantic purpose. I know it's hard to do that right now in the current system. We're working on color token updates right now that will include adding color spectrums so you'll have access to `grey-100`, `200`, etc for cases like this where it doesn't need a semantic mapping.

I would say go ahead and ship this and update the tokens when they are available. They'll ship in v11.

Reply:
Thank you for the heads up!